tests/unittests/tests-hash_string/Makefile.include

```diff
@@ -0,0 +1,2 @@
+USEMODULE += hash_string
+
```

Comment:
> @authmillenon : The reason I added the extra new line is because all the other one's already had an extra line for example tests-timex MakeFile.include , it is the same case with others like tests-crypto Makefile.include etc. So I intentionally added it here too.

No need to repeat another persons errors ;-).

> So should I remove it from here as well as others or just let it be here ?

Yes please remove it. You may remove them from the other files to, but please do that in another PR.